jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -523,23 +529,19 @@ private boolean isDDLCommand(int updatedCount, int columnCount) throws SQLExcept
       }
 
       if (character == '\\') {
-        antiSlash = true;
+        backslash = true;
       }
 
       if (character == '\'') {
-        if (antiSlash) {
-          antiSlash = false;
-        } else if (quoteString) {
```

Comment:
What I was trying to refer is, the above works, but another way to do the same is just by replacing this line with the above-mentioned changes.

Since I got the inspiration from https://github.com/postgres/pgadmin3/blob/794527d97e2e3b01399954f3b79c8e2585b908dd/pgadmin/dlg/dlgProperty.cpp#L999-L1045 and hence I'm of the opinion to make changes to a minimum.